packages/11ty/content/_assets/javascript/lit/lightbox.js

```diff
@@ -0,0 +1,342 @@
+import { LitElement, css, html } from 'lit';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+
+class Lightbox extends LitElement {
+  static properties = {
+    currentId: { attribute: 'current-id', type: String },
+    figures: {
+      converter: ((value) => value ? JSON.parse(decodeURIComponent(value)) : null)
+    },
+    height: { type: Number },
+    imageDir: { attribute: 'image-dir', type: String },
+    isInsideOpenModal: { attribute: 'is-inside-open-modal', type: Boolean },
```

Comment:
`isInsideOpenModal` prop is necessary to tell `lightbox` that it is inside a `modal` and render things differently. This seemed simpler than throwing out all of the nice encapsulation inside the shadow DOM